src/ensembl/src/shared/components/accordion/components/AccordionItemButton.tsx

```diff
@@ -34,7 +47,7 @@ AccordionItemButton.defaultProps = {
   extendDefaultStyles: true
 };
 
-type WrapperProps = Pick<
+type WrapperProps = { disabled?: boolean } & Pick<
   DivAttributes,
   Exclude<keyof DivAttributes, keyof InjectedButtonAttributes>
 >;
```

Comment:
The line below will say `const AccordionItemButtonWrapper: React.SFC<WrapperProps>`. `React.SFC` is a deprecated type (in favour of `FunctionComponent`), and the type annotation of `AccordionItemButtonWrapper` is likely unnecessary.

Reply:
P.S.: there are several examples of React.SFC in our codebase — all in accordion component. Could you please weed them out?

Reply:
That's done